packages/documentation/copy/en/reference/Utility Types.md

```diff
@@ -389,6 +391,8 @@ Released:
 
 Constructs a type consisting of the return type of function `Type`.
 
+For overloaded functions, this will be the return type of the _last_ signature; see [Inferring Within Conditional Types](docs/handbook/2/conditional-types.html#inferring-within-conditional-types).
```

Comment:
```suggestion
For overloaded functions, this will be the return type of the _last_ signature; see [Inferring Within Conditional Types](/docs/handbook/2/conditional-types.html#inferring-within-conditional-types).
```